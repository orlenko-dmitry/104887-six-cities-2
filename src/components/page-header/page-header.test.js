import React from 'react';
import {shallow} from 'enzyme';

import PageHeader from './page-header.jsx';

it(`PageHeader renders correctly`, () => {
  const tree = shallow(<PageHeader user={null} />);

  expect(tree).toMatchSnapshot();
});
