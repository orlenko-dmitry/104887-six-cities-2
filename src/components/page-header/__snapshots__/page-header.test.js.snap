// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders correctly 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <a
          className="header__logo-link"
          href="/"
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height={41}
            src="img/logo.svg"
            width={81}
          />
        </a>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <li
            className="header__nav-item user"
          >
            <Link
              className="header__nav-link header__nav-link--profile"
              to="/login"
            >
              <div
                className="header__avatar-wrapper user__avatar-wrapper"
              />
              <span
                className="header__login"
              >
                Sign in
              </span>
            </Link>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</header>
`;
